src/applications/survivors-benefits/config/chapters/02-claimant-information/claimantInformation.js

```diff
@@ -0,0 +1,78 @@
+import {
+  fullNameSchema,
+  titleUI,
+  ssnUI,
+  ssnSchema,
+  dateOfBirthSchema,
+  dateOfBirthUI,
+  yesNoSchema,
+  yesNoUI,
+} from 'platform/forms-system/src/js/web-component-patterns';
+import { seriouslyDisabledDescription } from '../../../components/formDescriptons/index';
+import { benefitsIntakeFullNameUI } from './helpers';
+
+/** @type {PageSchema} */
+export default {
+  title: 'Claimant information',
+  path: 'claimant/information',
+  uiSchema: {
+    ...titleUI("Claimant's relationship to the Veteran"),
+    claimantRelationship: {
+      'ui:title': "What is the claimant's relationship to the Veteran?",
+      'ui:widget': 'radio',
+      'ui:options': {
+        labels: {
+          SPOUSE: 'Surviving spouse',
+          CUSTODIAN: 'Custodian filing for child under 18',
+          ADULT_CHILD_STILL_IN_SCHOOL:
+            'Adult child who is 18-23 years old and still in school',
+          ADULT_CHILD_SERIOUSLY_DISABLED:
+            'Adult child who is seriously disabled',
+        },
+      },
+      'ui:errorMessages': {
+        required: 'Select what your relationship is to the Veteran',
+      },
+    },
+    seriouslyDisabled: {
+      'ui:description': seriouslyDisabledDescription,
```

Comment:
Since we are doing a file per page, I would recommend moving this into here and not importing from another file. It will make it easier to find things if everything for the page in is one place. If it shared, then maybe we will need to do something else.